.github/workflows/reusable_provenance.yaml

```diff
@@ -170,3 +170,58 @@ jobs:
         id: ent_upload
         run: |
           echo "provenance_digest=$(ent put ${{ needs.filename.outputs.filename }}.intoto.jsonl)" >> $GITHUB_OUTPUT
+
+  # Publishes a comment containing the digests of the generated artifact and its provenance.
+  publish_comments:
+    runs-on: ubuntu-20.04
+    needs: [upload_provenance, build_binary]
+    permissions:
+      # Allow the job to update the repo with the latest provenance info and index.
+      contents: write
+      # Allow the job to add a comment to the PR.
+      pull-requests: write
+
+    steps:
+      # Add the artifact digest and name to the ./comment file.
+      - name: Format artifact digest
+        run: |
+          echo -e "Artifact digest and name:\n\n\`\`\`" >> ./comment
+          echo "${{ needs.build_binary.outputs.digest }}" | base64 -d | tee -a ./comment
+          echo -e "\`\`\`" >> ./comment
```

Comment:
I hope we can find a nicer way of doing this at some point :)

Reply:
Yeah. Let me know if you have any ideas :)